### PR TITLE
Added GitHub actions

### PR DESCRIPTION
**Pull request by gabrieltemtsen**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -11,8 +11,8 @@ REDIS_URL="redis://localhost:6379"
 WORKER_CONCURRENCY="5"
 
 # Auto-target new signers
-TARGET_SIGNER_FID=
-TARGET_SIGNER_PUBKEY= # 0x...
+TARGET_SIGNER_FID=420564
+TARGET_SIGNER_PUBKEY=0xdD8eE5555744d6842e3c572e46F42092ed7326b6 # 0x...
```
Comment (stephancill) [inline .env.example]: remove pls

File: docker-compose.yml [hunk 2]
```diff
@@ -5,46 +5,58 @@ services:
...
+      timeout: 5s        # Allow up to 5 seconds for a response
+      retries: 3         # Fail after 3 unsuccessful attempts
+      start_period: 10s  # Start checks after 10 seconds
     networks:
       - indexer-network
+
   redis:
     image: 'redis:7.2-alpine'
     restart: unless-stopped
-    command: --loglevel warning --maxmemory-policy noeviction
```
Comment (stephancill) [inline docker-compose.yml]: is there a reason for removing these flags?

**Review by stephancill — CHANGES_REQUESTED** [review]
@gabrieltemtsen thank you for opening this PR! please see the comments and also remove the `dump.rdb` that i assume was accidentally committed

**Comment by stephancill** [comment]
@gabrieltemtsen do i just need to add the docker secrets or is there something else?

File: docker-compose.yml (hunk 2 above)
Comment (gabrieltemtsen) [reply docker-compose.yml]: Oh no, I was troubleshooting and it got left out

**Comment by gabrieltemtsen** [comment]
You just need to add your docker - Username and password 
as the secret for the actions to work properly

**Review by stephancill — APPROVED** [review]

**Pull request merged**